src/js/lib/monster.ui.js

```diff
@@ -741,67 +741,127 @@ define(function(require) {
 		},
 
 		dialog: function(content, options) {
-			var isPersistent = _.has(options, 'isPersistent')
-				? options.isPersistent
-				: false;
-			var dialog = $('<div />').append(content),
-				coreApp = monster.apps.core,
-				i18n = coreApp.i18n.active(),
-				dialogType = typeof options !== 'undefined' && typeof options.dialogType !== 'undefined' ? options.dialogType : 'classic',
-				closeBtnText = i18n.close || 'X';
-
-			if (typeof options !== 'undefined' && typeof options.dialogType !== 'undefined') {
-				delete options.dialogType;
-			}
-
-			// delete options.dialogType;
-			$('input', content).keypress(function(e) {
-				if (e.keyCode === 13) {
-					e.preventDefault();
-					return false;
-				}
+			// Get options
+			var dialogType = _.get(options, 'dialogType', 'classic');
+			var hideClose = _.get(options, 'hideClose', false);
+			var isPersistent = _.get(options, 'isPersistent', false);
+			var onClose = _.get(options, 'onClose');
+			var open = _.get(options, 'open');
+			var scrollableContainer = _.get(options, 'scrollableContainer');
```

Comment:
The vertical scroll works as expected for the edit device dialog (the horizontal one is set to the dialog body instead of the container). Can I keep it for this case, and leave it undocumented so other devs do not use it? (For the moment I have removed the parameter at commit 7cb48502a8a821e3e5fc7aa77c85db3d9adcea16, but let me know your thoughts)

![image](https://user-images.githubusercontent.com/893877/57310581-46bab000-70a7-11e9-9589-fe9b5706ff23.png)

Reply:
That's only one use case though, I've seen a couple of others where the dialog's height is changed programmatically after render and overflows the viewport.

I think it is fine for now and we can revisit this feature latter.